### PR TITLE
Demo App - Show Apple Pay button when Apple Pay is selected

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

 - Update demo app to show the Apple Pay button when Apple Pay is selected in Drop-in. Now, tapping on the Apple Pay button will present the Apple Pay sheet, whereas previously tapping the "Complete Purchase" button presented the Apple Pay sheet (which violates Apple's guidelines). We've seen some confusion from merchants around the right way to present the Apple Pay sheet, so now we'll be able to point them to our demo app for a working example.
- Clarify Apple Pay instructions in README

Additional clean-up:
 - Restructure demo app so that we don't have as much nesting of files in different folders.
 - Update demo app settings
     - Remove unused options for modal presentation and coinbase
     - Move color scheme option to settings & remove segmented control

 ### Checklist

 - ~[ ] Added a changelog entry~

- @sestevens 

### Screenshots

**New file structure**

Screen Shot 2021-03-26 at 2 58 21 PM

| Demo app with PayPal selected | Demo app with Apple Pay selected |
|-------|--------|
|Simulator Screen Shot - iPhone 12 - 2021-03-26 at 15 00 24| Simulator Screen Shot - iPhone 12 - 2021-03-26 at 15 00 31|

### CODE REVIEW

**Review by sarahkoop — APPROVED** [review]
Looks good to me - should there be any UI tests added for this change?

**Comment by sestevens** [comment]
> Looks good to me - should there be any UI tests added for this change?

Since it's a change to the demo app, not the Drop-in itself, I think we're OK without UI tests.

File: README.md [hunk 1]
```diff
@@ -93,7 +93,12 @@ let canMakePayments = PKPaymentAuthorizationViewController.canMakePayments() &&
...
+**Important:** If your customer selects Apple Pay as their preferred payment method then `result.paymentOptionType == .applePay` and the `result.paymentMethod` will be `nil`. Selecting Apple Pay does not display the Apple Pay sheet or create a nonce. After you receive the `BTDropInResult`, you will need to:
+1) Display a `PKPaymentButton`.
+2) Present a `PKPaymentAuthorizationViewController` when the customer taps the button.
+3) Tokenize the `PKPayment` and send the resulting nonce to your server to create a transaction.
+
+Use `BTApplePayClient` to tokenize the customer's Apple Pay information - [view our official docs for more information](https://developers.braintreepayments.com/guides/apple-pay/client-side/ios/v5).
```
Comment (scannillo) [inline README.md:101]: 👏

**Review by scannillo — APPROVED** [review]
Changes look good to me

**Pull request merged**